tests/unit/test_clients.py

```diff
@@ -85,7 +85,7 @@ def test_get_oracle_data_client():
         clients.get_data_client(ORACLE_CONN_CONFIG)
 
 
-def test_get_pandas_data_client(fs):
+def test_get_pandas_data_client():
```

Comment:
is this just cleanup or was the test failing?

Reply:
I saw an error one time saying fs is not a valid fixture. I can no longer reproduce it. Therefore consider it a cleanup.